ic-os/guestos/context/Dockerfile

```diff
@@ -96,6 +96,7 @@ RUN \
 RUN systemctl enable \
     chrony \
     nftables \
+    ssh \
```

Comment:
was this already enabled before?

Reply:
In 24.04, it seems ssh is no longer enabled by default.